src/main/java/com/google/sps/servlets/StoreVideos.java

```diff
@@ -0,0 +1,64 @@
+package com.google.sps.servlets;
+
+import com.google.sps.data.Video;
+import java.util.ArrayList;
+import com.google.gson.Gson;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Key;
+import com.google.appengine.api.datastore.KeyFactory;
+import com.google.appengine.api.datastore.EntityNotFoundException;
+import com.google.appengine.api.datastore.Transaction;
+import java.io.IOException;
+import java.util.ConcurrentModificationException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class StoreVideos {
+
+  DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+
+  public Entity incrementSearchCount(String id, float sentiment)
+      throws EntityNotFoundException, ConcurrentModificationException {
+    Key videoKey = KeyFactory.createKey("Video", id);
+    Entity videoEntity = datastore.get(videoKey);
+    Long numSearches = (long) videoEntity.getProperty("numSearches");
+    videoEntity.setProperty("numSearches", numSearches + 1);
+    return videoEntity;
+  }
+
+  public Entity createVideoEntity(String id) {
+    Entity videoEntity = new Entity("Video", id);
+    videoEntity.setProperty("sentiment", 0);
+    videoEntity.setProperty("numSearches", 1);
+    return videoEntity;
+  }
+  
+  public void addToDatabase(String id, float sentiment) {
```

Comment:
this function will be called by Sentiment Servlet @kayefung in a later PR because it was a lot for one PR. The two are pretty disconnected though because they aren't really that related I guess, getting from the database requires a user request, but we want to store a video search whenever a search happens in index.html